index.html

```diff
@@ -857,11 +857,17 @@ <h4>Text</h4>
                 <a>Text</a> objects in one language can also contain some words in a different language.
                 This is represented in a <a>DAPT Document</a> by setting the <code>xml:lang</code> attribute on
                 inner <code>&lt;span&gt;</code> elements.</p>
+                <p class="note"><code>&lt;span&gt;</code> elements can be used to add specific styling (e.g. Japanese ruby annotations).</p>
+                <p class="note"><code>&lt;span&gt;</code> elements can be used to add specific timing
+                as illustrated in [[[#example-10]]] to indicate the timing of the audio rendering
+                of the relevant section of text. Per [[TTML2]], timing of the <code>&lt;span&gt;</code>
+                element is relative to the parent element's computed begin time.</p>
+                <p class="note">The text content of a <a>Text</a> object is the concatenated text content
+                of the <code>&lt;p&gt;</code> element and its descendent <code>&lt;span&gt;</code> elements.</p>
```

Comment:
I think (feel free to check) that it's the _character_ content of the `<p>` element and its descendent `<span>`s - also should we mention white space handling here, since typically consecutive white space is collapsed?

Reply:
Yes, we should mention white space handling, at least to say that the TTML semantics apply.

Reply:
In [XML1.0](https://www.w3.org/TR/xml/), "character content" is only mentioned once, with no definition. 
In [TTML2](https://www.w3.org/TR/2018/REC-ttml2-20181108/), it is used 5 times but not linked to any specific definition.
It wasn't clear to me why I proposed it as a note. I now propose to rephrase the first bullet point in this list.

Reply:
I think this is the right term though. For example it is used in XPath Data Model in the overview of [Text Nodes](https://www.w3.org/TR/xpath-datamodel-31/#TextNode):
>Text Nodes encapsulate XML character content.

Reply:
I changed the text to use "character content"